### PR TITLE
Add streamlabs-obs 0.1.0

### PR DESCRIPTION
**Pull request by ngdio**
Streamlabs OBS is a branded version of OBS Studio, containing additional features. Mac version was just released.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/streamlabs-obs.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'streamlabs-obs' do
+  version '0.1.0'
+  sha256 '019f71588d710fea781340db71a9a229bbd35ec51a7a56687f4fa772f25c1afe'
+
+  url "https://slobs-cdn.streamlabs.com/Streamlabs+OBS-#{version}-mac.dmg"
+  appcast 'https://github.com/stream-labs/streamlabs-obs/releases.atom'
```
Comment (vitorgalvao) [inline Casks/streamlabs-obs.rb:6]: Not a good appcast. Does not contain the version number.

Comment (vitorgalvao) [reply Casks/streamlabs-obs.rb:6]: Use `https://slobs-cdn.streamlabs.com/latest-mac.yml`. It was found by the script linked on the appcast documentation page.

Comment (ngdio) [reply Casks/streamlabs-obs.rb:6]: It is though? Not different from other appcasts I've seen here.
```
<entry>
    <id>tag:github.com,2008:Repository/108469351/v0.1.0-mac</id>
    <updated>2020-03-30T22:37:47+02:00</updated>
    <link rel="alternate" type="text/html" href="https://github.com/stream-labs/streamlabs-obs/releases/tag/v0.1.0-mac"/>
    <title>v0.1.0-mac</title>
    <content type="html">&lt;p&gt;Release 0.1.0-mac&lt;/p&gt;</content>
    <author>
      <name>andycreeth</name>
    </author>
    <media:thumbnail height="30" width="30" url="https://avatars3.githubusercontent.com/u/3180181?s=60&amp;v=4"/>
  </entry>
```

**Comment by cweagans** [comment]
🎉  thanks for merging this one so quick @vitorgalvao! And thanks for building the cask @ngdio! Really glad to have this available.

**Pull request merged**